.github/workflows/build.yml

```
@@ -131,7 +131,7 @@ jobs:
           target_dir: ${{ needs.smoke-tests.outputs.OUTPUT_DIR }}
           git_branch: ${{ needs.smoke-tests.outputs.GIT_BRANCH }}
           git_tag: ${{ needs.smoke-tests.outputs.GIT_TAG }}
-          event_name: ${{ github.event_name }}
+          deploy: ${{ github.event_name == 'push' && github.event.pull_request.head.repo.full_name == 'org/repo' }} # Only deploy if this is a push event, not a PR event and if this is not a PR originating from a fork
```

I'm not sure if `github.event_name == 'push'` is enough to trigger the deploy from PRs of the FTL repo or if it would require a (foreced) push after a PR has been opened. Maybe we need to have
```suggestion
          deploy: ${{ (github.event_name == 'push' || github.event_name == 'pull_request') && github.event.pull_request.head.repo.full_name == 'org/repo' }} # Only deploy if this is a push event, not a PR event and if this is not a PR originating from a fork
```